Makefile

```diff
@@ -0,0 +1,72 @@
+clean:
+	rm -rf bazel-*
+	rm -rf man
+	rm -rf docs
+
+build:
+	bazel build //...
+
+test:
+	bazel test //...
+
+.PHONY: clean build test
+
+bazel-bin/verilog/tools/lint/verilog_lint:
+	bazel build //verilog/tools/lint:verilog_lint
+
+bazel-bin/verilog/tools/syntax/verilog_syntax:
+	bazel build //verilog/tools/syntax:verilog_syntax
+
+bazel-bin/verilog/tools/formatter/verilog_format:
+	bazel build //verilog/tools/formatter:verilog_format
+
+tools: bazel-bin/verilog/tools/lint/verilog_lint bazel-bin/verilog/tools/syntax/verilog_syntax bazel-bin/verilog/tools/formatter/verilog_format
+	@true
+
+# Generate man pages from the tools
+man/verilog_lint.1: bazel-bin/verilog/tools/lint/verilog_lint
+	mkdir -p man
+	gflags2man --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/lint/verilog_lint
+
+man/verilog_syntax.1: bazel-bin/verilog/tools/syntax/verilog_syntax
+	mkdir -p man
+	gflags2man --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/syntax/verilog_syntax
+
+man/verilog_format.1: bazel-bin/verilog/tools/formatter/verilog_format
+	mkdir -p man
+	gflags2man  --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/formatter/verilog_format
```

Comment:
If we submit this, we should then also update the documentation that `gflags2man` is a requirement for `make install`